src/router_core/connections.c

```diff
@@ -1456,42 +1456,48 @@ void qdr_process_addr_attributes_CT(qdr_core_t *core, qdr_address_t *addr)
 static void qdr_connection_group_setup_CT(qdr_core_t *core, qdr_connection_t *conn)
 {
     const char *correlator = conn->connection_info->group_correlator;
+    uint64_t    ordinal    = conn->connection_info->group_ordinal;
 
     assert(conn->role == QDR_ROLE_INTER_ROUTER);
     qd_log(LOG_ROUTER_CORE, QD_LOG_DEBUG,
-           "CGROUP qdr_connection_group_setup_CT - %lx (%s)",
-           (ulong) conn, conn->connection_info->host);
+           "CGROUP setup group '%s' for parent [C%"PRIu64"] with ordinal=%"PRIu64" (host=%s)",
```

Comment:
```suggestion
           "Connection group setup group '%s' for parent [C%"PRIu64"] with ordinal=%"PRIu64" (host=%s)",
```

Reply:
People might confuse CGROUP with the Linux cgroup

Reply:
Good point - I should've thought of that!

Reply:
Done.